web/beacon-app/src/components/layout/AppLayout.tsx

```diff
@@ -28,6 +29,9 @@ function AppLayout({ children, Breadcrumbs }: PageProps) {
   return (
     <>
       <Topbar Breadcrumbs={Breadcrumbs} isOnboarded={isOnboarded} profileData={loaderData} />
+      {/* TODO: Display SandboxBanner only to user's with the sandbox account type. 
```

Comment:
A condition will be added in a future story to display this component to the correct user.